### PR TITLE
Tar.gz to tar

### PR DESCRIPTION
**Pull request by HuangJiameng**
See https://github.com/deepmodeling/dpgen/issues/766
- change .tar.gz to .tar, changed tar -zcf to tar -cf,

### CODE REVIEW

File: dpdispatcher/hdfs_context.py [hunk 1]
```diff
@@ -130,11 +130,11 @@ def download(self,
...
         os.chdir(gz_dir)
-        rfile_tgz = "%s/%s_*_download.tar.gz" % (self.remote_root, submission.submission_hash)
+        rfile_tgz = "%s/%s_*_download.tar" % (self.remote_root, submission.submission_hash)
         lfile_tgz = "%s/tmp/" % (self.local_root)
         HDFS.copy_to_local(rfile_tgz, lfile_tgz)
 
-        tgz_file_list = glob(os.path.join(self.local_root, "tmp/*_download.tar.gz"))
+        tgz_file_list = glob(os.path.join(self.local_root, "tmp/*_download.tar"))
         for tgz in tgz_file_list:
             with tarfile.open(tgz, "r:gz") as tar:
```
Comment (njzjz) [inline dpdispatcher/hdfs_context.py:139]: The gzip option `gz` still exists.

**Comment by njzjz** [comment]
This may not improve performance if the Internet speed or the I/O speed is the bottleneck. Compared with the time spent transferring large files, the compression time is insignificant. I suggest adding an option instead to disable compression.

File: dpdispatcher/hdfs_context.py (hunk 1 above)
Comment (HuangJiameng) [reply dpdispatcher/hdfs_context.py:139]: ok I see

**Comment by njzjz** [comment]
Also, note that this PR only modifies HDFS context. Most people  use SSH context.

**Comment by HuangJiameng** [comment]
> I suggest adding an option instead to disable compression.
Do you mean an option to let the user choose whether compress with -cf or -cvf?

**Comment by njzjz** [comment]
> > I suggest adding an option instead to disable compression.
>
>  Do you mean an option to let the user choose whether compress with -cf or -cvf?

Yes.. We may not break the current behavior

**Comment by HuangJiameng** [comment]
[https://github.com/deepmodeling/dpgen/issues/766#issuecomment-1176792502](url)
so you mean an option to skip compression process or to change the way of compression?

**Comment by njzjz** [comment]
> [https://github.com/deepmodeling/dpgen/issues/766#issuecomment-1176792502](url) so you mean an option to skip compression process or to change the way of compression?

Yes, like the option `clean_asynchronously`, although I don't think it's urgent.

**Pull request closed without merging**